employee_project/employee.py

```diff
@@ -68,3 +73,16 @@ def welcome_page():
     print(" Exit the Apllication Enter number           : 4")
     choice = int(input("\nEnter your choice : "))
     return choice
+
+
+def get_salary(msg = "Enter employee salary:"):
+    """ @param: string
+        @desc: Receive the input from the user and validating the data
+        @returns: int
+    """
+    try:
+        inp = int(input(msg))
```

Comment:
inp variable name,  change to sal (Every variable name should be a meaningful )

Reply:
changed